server/start.go

```diff
@@ -101,6 +101,14 @@ which accepts a path for the resulting pprof file.
 				return startStandAlone(serverCtx, appCreator)
 			}
 
+			serverCtx.Logger.Info("Unlocking keyring")
+
+			// fire unlock precess for keyring
+			_, err = clientCtx.Keyring.List()
```

Comment:
sorry, I don't understand how does this fix the issue. Calling `List` unlocks the other backends?

Reply:
so do you have to input the password when the server starts? Can you add an `if` and only call `List` if the keyring backend is `file` instead?

Reply:
@davcrypto bump

Reply:
Thanks for your advise. Just updated.

Reply:
@fedekunze bump